pkg/detector/policy.go

```diff
@@ -270,3 +270,40 @@ func (d *ResourceDetector) removeResourceLabelsIfNotMatch(objectReference workv1
 	}
 	return true, nil
 }
+
+func (d *ResourceDetector) getMatchedResourceBindings(policyNamespace, policyName string) (*workv1alpha2.ResourceBindingList, error) {
+	var ls labels.Set
+	if len(policyNamespace) == 0 {
+		ls = labels.Set{policyv1alpha1.ClusterPropagationPolicyLabel: policyName}
+	} else {
+		ls = labels.Set{
+			policyv1alpha1.PropagationPolicyNamespaceLabel: policyNamespace,
+			policyv1alpha1.PropagationPolicyNameLabel:      policyName,
+		}
+	}
```

Comment:
Please think about how to test it.
I'm thinking if we can have functions like `ListPPDerivedRB` `ListCPPDerivedCRB`.

Reply:
+1

Reply:
Done